src/backend/elfobj.c

```diff
@@ -463,13 +462,15 @@ static IDXSTR elf_addmangled(Symbol *s)
  *      sz      =       symbol size
  *      typ     =       symbol type
  *      bind    =       symbol binding
- *      segidx  =       segment index for segment where symbol is defined
+ *      sec     =       index of section where symbol is defined
+ *      vsblty  =       visibility of symbol
```

Comment:
I hate to complain about names, but why not just call it `visibility`?

Also, what is visibility of names? You could add `STV_xxxx` to the explanatory comment.